Cython/Compiler/Nodes.py

```diff
@@ -1769,10 +1775,27 @@ def analyse_declarations(self, env, enum_entry):
             self.value, self.pos, cname=cname,
             visibility=enum_entry.visibility, api=enum_entry.api,
             create_wrapper=enum_entry.create_wrapper and enum_entry.name is None)
+
+        enum_value = None
+        if self.value:
+            if self.value.is_literal:
+                enum_value = int(self.value.value)
+            else:
+                value_entry = getattr(self.value, 'entry', None)
```

Comment:
It would feel better to spell out the cases where `.entry` exists, rather than having it available *sometimes*. It's unclear from the code what cases those are.